### PR TITLE
add list of references

### PR DESCRIPTION
**Pull request by gavinking**

### CODE REVIEW

File: documentation/src/main/asciidoc/reference/introduction.adoc [hunk 1]
```diff
@@ -17,6 +17,18 @@ simplistic example program in the {example}[`session-example`] directory,
...
+
+This document assumes some passing familiarity with Hibernate ORM or another
+implementation of JPA. If you've never used JPA before, that's OK, but you
+might need to refer to the following sources of information at some points
+in this text:
+
+- the http://hibernate.org/orm/documentation/5.5/[documentation for Hibernate ORM],
+- the https://jcp.org/aboutJava/communityprocess/mrel/jsr338/index.html[JPA 2.2 specification], or
+- https://www.manning.com/books/java-persistence-with-hibernate-second-edition[Java Persistence with Hibernate],
+  the third edition of the original book _Hibernate in Action_.
```
Comment (DavideD) [inline documentation/src/main/asciidoc/reference/introduction.adoc:30]: You say third edition but it's actually the second edition

**Review by DavideD — COMMENTED** [review]
Looks good, I've just left a comment about second or third book edition

File: documentation/src/main/asciidoc/reference/introduction.adoc (hunk 1 above)
Comment (gavinking) [reply documentation/src/main/asciidoc/reference/introduction.adoc:30]: It's the 3rd edition of "Hibernate in Action". It's the second named "Java Persistence with Hibernate". Confusing, I know.

Comment (DavideD) [reply documentation/src/main/asciidoc/reference/introduction.adoc:30]: Can't we just not mentioned the exact edition and say `the updated version of the original book HIbernate in Action`? Or something similar

Comment (gavinking) [reply documentation/src/main/asciidoc/reference/introduction.adoc]: If you insist.

Comment (DavideD) [reply documentation/src/main/asciidoc/reference/introduction.adoc]: Yes, please

**Comment by DavideD** [comment]
Merged https://github.com/hibernate/hibernate-reactive/pull/922,  thanks

**Pull request merged**